lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View Teams in the Wind Energy Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+       <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Topic</p>
+              <h3 class="card-title">Renewable Energy</h3>
+              <p class="card-description">Explore how changes in renewable energy production will change our day-to-day lives</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper has-actions">
+            <div class="card-meta-details">
+              <p class="meta-detail team-count">34 Teams</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View Teams in the Renewable Energy Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Topic</p>
+              <h3 class="card-title">Solar Power Cells</h3>
+              <p class="card-description">Discuss the current technology trajectory of solar power cells</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper has-actions">
+            <div class="card-meta-details">
+              <p class="meta-detail team-count">34 Teams</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View  Teams in the Solar Power Cells Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Topic</p>
+              <h3 class="card-title">Wind Energy</h3>
+              <p class="card-description">Explore how we can harness wind energy, either through large turbine farms, mid-ocean wind farms, and other types of wind energy that would be cut off from this listing because it is so long</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper has-actions">
+            <div class="card-meta-details">
+              <p class="meta-detail team-count">34 Teams</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View Teams in the Wind Energy Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+        <hr />
+        <br />
```

Comment:
I'm cool with the ``<hr />`` element, but can you remove the ``<br />`` element? - it shouldn't be needed or included in spacing these guys out in an actual layout